aalto/aaltodata.rst

```diff
@@ -126,9 +126,12 @@ everyone in the group has access.
       there are fixed multi-year projects, they can also get a group.
 
 -  Groups are managed by IT staff. To request a group, mail us with
-   the necessary information (see next point).
+   the necessary information (see bottom of page).
 -  Each group has an owner, quota on filesystems, and some other
-   metadata (see below),
+   metadata (see below).
+-  Group membership is per-account, not tied to employment contracts.
+   If you want someone to lose access, you have to explicitly remove
```

Comment:
If you want someone to lose access to a group you manage, you have to explicitly ask the administrators to remove the account from your group (group owners at CS dept can do this themselves).